foyer/forcefield.py

```diff
@@ -308,6 +308,15 @@ def apply(self, topology, references_file=None, *args, **kwargs):
         references_file : str, optional, default=None
             Name of file where force field references will be written (in Bibtex
             format)
+        use_residue_map : boolean
```

Comment:
A couple nitpicks in this docstring. Could you make this `use_residue_map : boolean, optional, default=True`?